ansible_collections/arista/avd/plugins/plugin_utils/topology_generator_utils.py

```diff
@@ -0,0 +1,715 @@
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+import os
+import queue
+import math
+
+import yaml
+import textwrap
+
+import drawSvg as draw
+
+# MARGINS between real groups
+BOXMARGIN = 25
+
+# Sizings
+ROUTERMAXFONTSIZE = 25
+ROUTERWRAPFONTSIZE = 18
+ROUTERSIZE = 120
+ROWSPACING = ROUTERSIZE * 2
+LVLSPACING = ROUTERSIZE * 2
+PORTHEIGHT = 30
+PORTWIDTH = 20
+PORTOFFSET = 3
+PORTFONTSIZE = 16
+TITLEFONTSIZE = 24
+TITLEOFFSET = 0
+FONTWIDTHHEIGHTRATIO = 0.62
+
+# Used for horizontal connection overlaps
+WIREDEFAULTDISTANCE = 10
+WIRESPREAD = 6
+
+EDGETYPES = ["server"]
+
+def read_yaml_file(filename):
+    """
+    Open a structure config file and load a data from the file
+
+    Args:
+        filename: Name of the file
+
+    Returns:
+        Data from the file
+    """
+    with open(filename, "r", encoding="utf-8") as stream:
+        data = None
+        try:
+            data = yaml.safe_load(stream)
+        except yaml.YAMLError as exc:
+            print(exc)
```

Comment:
don't use `print` for ansible modules rather import display as in `verify_requirements` and use `display.error`